content/chainguard/chainguard-images/migration-guides/debian-compatibility.md

```diff
@@ -0,0 +1,374 @@
+---
+title: "Debian Compatibility"
+linktitle: "Debian Compatibility"
+type: "article"
+description: "Differences between Chainguard Images and Debian third-party images"
+date: 2024-02-08T15:56:52-07:00
+lastmod: 2024-02-08T15:56:52-07:00
+draft: false
+tags: ["IMAGES", "PRODUCT", "CONCEPTUAL"]
+images: []
+menu:
+  docs:
+    parent: "concepts"
+weight: 600
+toc: true
+---
+
+Chainguard Images and Debian base images have different binaries and scripts included in their respective `busybox` and `coreutils` packages.
+
+The following table lists common tools and their corresponding package(s) in both Wolfi and Debian distributions.
+
+Note that `$PATH` locations like `/usr/bin` or `/sbin` are not included here. If you have compatibiltiy issues with tools that are included in both `busybox` and `coreutils`, be sure to check `$PATH` order and confirm which version of a tool is being run.
+
+Generally, if a tool exists in `busybox` but does not have a `coreutils` counterpart, there will be a specific package that includes it. For example the `zcat` utility is included in the `gzip` package in both Wolfi and Debian.
+
+You can use the `apk search` command in Wolfi, and the `apt-cache search` command in Debian to find out which package includes a tool.
+
+| Utility                      | Wolfi busybox | Debian busybox | Wolfi coreutils | Debian coreutils |
+| ---------------------------- | ------------- | -------------- | --------------- | ---------------- |
```

Comment:
```suggestion
| ---------------------------- | :-----------: | :-----------: | :-----------: | :-----------: |
```
centering these columns might make the table a bit more readable, not 100% sure

Reply:
Yeah I like that too @SharpRake. Also, I have never used Markdown alignment and you just blew my mind, how am I just learning about this today? 🤯